x-pack/plugins/synthetics/common/runtime_types/monitor_management/monitor_types_project.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as t from 'io-ts';
+import { ScreenshotOptionCodec } from './monitor_configs';
+
+export const ProjectMonitorThrottlingConfigCodec = t.interface({
+  download: t.number,
+  upload: t.number,
+  latency: t.number,
+});
+
+export const ProjectBrowserMonitorCodec = t.intersection([
+  t.interface({ id: t.string, name: t.string, schedule: t.number, content: t.string }),
+  t.partial({
+    locations: t.array(t.string),
+    throttling: ProjectMonitorThrottlingConfigCodec,
+    screenshots: ScreenshotOptionCodec,
+    tags: t.array(t.string),
+    ignoreHTTPSErrors: t.boolean,
+    apmServiceName: t.string,
+    playwrightOptions: t.record(t.string, t.unknown),
+    filter: t.interface({
+      match: t.string,
+    }),
+    params: t.record(t.string, t.unknown),
+    enabled: t.boolean,
+  }),
+]);
+
+export const ProjectMonitorsRequestCodec = t.interface({
+  project: t.string,
```

Comment:
```suggestion
  projectId: t.string,
```

Reply:
It's project from the synthetics agent side. I could ask @vigneshshanmugam to change it. I prefer using the `Id` postfix for things, but I didn't want to enforce that on the synthetics agent and Vignesh.